src/config/serverConfig.ts

```diff
@@ -1,5 +1,29 @@
-import { constants, ServerConfig } from '@projecttacoma/node-fhir-server-core';
+import { constants, ServerConfig, resolveSchema } from '@projecttacoma/node-fhir-server-core';
 import { MeasureService, LibraryService } from '../services';
+import capabilityStatementResources from './capabilityStatementResources.json';
+
+const customCapabilityStatement = (): fhir4.CapabilityStatement => {
+  const base_version = constants.VERSIONS['4_0_1'];
+  const CapabilityStatement = resolveSchema(base_version, 'CapabilityStatement');
+
+  return new CapabilityStatement({
+    id: 'MeasureRepositoryServiceCapabilityStatement',
+    name: 'MeasureRepositoryService',
+    title: 'FHIR Measure Repository Service Capability Statement',
+    status: 'active',
+    // date last modified
+    date: '2023-01-27',
+    publisher: 'The MITRE Corporation',
+    instantiates: [
+      'http://hl7.org/fhir/us/cqfmeasures/CapabilityStatement/shareable-measure-repository',
+      'http://hl7.org/fhir/us/cqfmeasures/CapabilityStatement/publishable-measure-repository'
```

Comment:
publishable-measure-repository capability statement looks like it supports read and search for MeasureReport objects, which the measure repository service doesn't do yet. Should we hold off on adding this until we add read and search capability for MeasureReports?

Reply:
My understanding of the publishable MR is that everything MeasureReport-related is optional since they all appear as `MAY` (ex. "MAY support retrieval of test cases by server-specific id through the MeasureReport/read interaction"), which would mean that we do instantiate the publishable MR